contracts/governance/FeeSharingCollector/FeeSharingCollector.sol

```diff
@@ -119,6 +131,40 @@ contract FeeSharingCollector is
     /// @dev fallback function to support rbtc transfer when unwrap the wrbtc.
     function() external payable {}
 
+    /**
+     * @dev initialize function for fee sharing collector proxy
+     * @param wrbtcToken wrbtc token address
+     * @param loanWrbtcToken address of loan token wrbtc (IWrbtc)
+     */
+    function initialize(address wrbtcToken, address loanWrbtcToken) external onlyOwner {
+        setWrbtcToken(wrbtcToken);
+        setLoanTokenWrbtc(loanWrbtcToken);
+    }
+
+    /**
+     * @notice Set the wrbtc token address of fee sharing collector.
+     *
+     * only owner can perform this action.
+     *
+     * @param newWrbtcTokenAddress The new address of the wrbtc token.
+     * */
+    function setWrbtcToken(address newWrbtcTokenAddress) public onlyOwner {
+        emit SetWrbtcToken(msg.sender, wrbtcTokenAddress, newWrbtcTokenAddress);
+        wrbtcTokenAddress = newWrbtcTokenAddress;
+    }
+
+    /**
+     * @notice Set the loan wrbtc token address of fee sharing collector.
+     *
+     * only owner can perform this action.
+     *
+     * @param newLoanTokenWrbtcAddress The new address of the loan wrbtc token.
+     * */
+    function setLoanTokenWrbtc(address newLoanTokenWrbtcAddress) public onlyOwner {
+        emit SetLoanTokenWrbtc(msg.sender, loanTokenWrbtcAddress, newLoanTokenWrbtcAddress);
+        loanTokenWrbtcAddress = newLoanTokenWrbtcAddress;
```

Comment:
should be a contract address, cannot be 0

Reply:
Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/519/commits/d3e520f0166c70d672d645ac02824662abc1c874)